pkg/plugin/test-keeper/plugin/test_checker.go

```diff
@@ -12,23 +12,48 @@ type TestChecker struct {
 	TestKeeperMatcher TestMatcher
 }
 
-// IsAnyNotExcludedFileTest checks if a commit affects any test file
-func (checker *TestChecker) IsAnyNotExcludedFileTest(files []scm.ChangedFile) (bool, error) {
+// FileCategories holds information about the total files coming in the changeset, legit files (those which are excluded from test verification)
+// and Tests
+type FileCategories struct {
+	Total, Legit, Tests int
+	Files               *[]scm.ChangedFile
+}
+
+// OnlyLegitFiles indicates if changeset contains only files which are excluded from test verification
+func (f *FileCategories) OnlyLegitFiles() bool {
```

Comment:
aren't they more likely excluded or skipped files? Legit is (for me) something very vague and general.

Reply:
Skipped is a good name! Thanks

Reply:
e5df495